src/market-programs.ts

```diff
@@ -1076,4 +1076,19 @@ export const marketPrograms: MarketRewardProgramArgs[] = [
     },
     chainId: ChainId.MAINNET,
   },
-];
+  // cbBTC/EURC 12,500 EURC 09/24/2024 11/08/2024 1pm EST
```

Comment:
Please update comment here to reflect September 26, 2024 1:00:00 PM as start of incentives